troubleshooting/known_issues/enable_iscsid_multipathd.md

```diff
@@ -0,0 +1,230 @@
+# Boot content projection services can fail if `iscsid` and `multipathd` services are not enabled on compute nodes and UANs
+
+## Issue Description
+
+iSCSI based boot content projection which is also known as "Scalable Boot Content Projection" (SBPS) for `rootfs` and `PE` images
+is supported in CSM version CSM 1.6.0 and later. On a customer system, using `CSM-1.6.0` with `USS-1.1.x` on compute nodes/ UANs in order to support AARCH64 images,
+`iscsid` and `multipathd` services are not enabled by default. SBPS will not be resilient across worker node reboots if these services are not enabled by default on compute nodes or UANs.
+
+## Issue Identification
+
+This issue can be identified by the following symptoms:
+
+On a compute node or UAN (iSCSI Initiator) we can observe the following SQUASHFS error messages in the console log:
+
+```bash
+dmesg -T | grep "SQUASHFS error" | head -n  1
+```
+
+Example output:
+
+```text
+[Sat Nov  2 22:32:41 2024] SQUASHFS error: xz decompression failed, data probably corrupt
+```
+
+On a compute node or UAN (iSCSI Initiator) we can observe that the `iscsid` service is not active:
+
+```bash
+ncn-s004# systemctl status iscsid
```

Comment:
The prompt is for a storage node but the preceding text says "on a compute or UAN."